### PR TITLE
41 make app

### PR DESCRIPTION
**Pull request by blazejpsz**
Create app - music

### CODE REVIEW

File: music_collection/manage.py [hunk 1]
```diff
@@ -20,3 +20,4 @@ def main():
 
 if __name__ == '__main__':
     main()
+    print("hello world")
```
Comment (nataliacza) [inline music_collection/manage.py:23]: I guess this is just a test line?

File: music_collection/music_collection/settings.py [hunk 2]
```diff
@@ -43,6 +43,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    "music",
```
Comment (nataliacza) [inline music_collection/music_collection/settings.py:46]: Single quote would be better here I guess.

**Review by nataliacza — CHANGES_REQUESTED** [review]

**Review by KateuszMacprzak — APPROVED** [review]

File: music_collection/music_collection/settings.py (hunk 2 above)
Comment (Lewer95) [reply music_collection/music_collection/settings.py:46]: I agree with u :)

**Review by Lewer95 — APPROVED** [review]
Looks fine for me :)

**Review by KN-9 — APPROVED** [review]

**Review by nataliacza — APPROVED** [review]
All good

**Review by nataliacza — APPROVED** [review]

**Pull request merged**